hack/generate-manifest.sh

```diff
@@ -201,8 +201,9 @@ if [ "$TUN_TYPE" != "geneve" ] && [ "$TUN_TYPE" != "vxlan" ] && [ "$TUN_TYPE" !=
     exit 1
 fi
 
-if [ "$MODE" == "release" ] && [ -z "$IMG_NAME" ]; then
-    echoerr "In 'release' mode, environment variable IMG_NAME must be set"
+if [ "$MODE" == "release" ] && [ -z "$AGENT_IMAGE" ] && [ -z "$CONTROLLER_IMAGE" ]; then
```

Comment:
nit: maybe keep the `IMG` spelling for image, for consistency with the existing `IMG_TAG` variable
so `AGENT_IMG_NAME` and `CONTROLLER_IMG_NAME`?